AVS-Landing-Zone/GreenField/Bicep/ESLZDeploy.bicep

```diff
@@ -54,7 +52,7 @@ module AVSCore 'Modules/AVSCore.bicep' = {
   name: '${deploymentPrefix}-AVS'
   params: {
     Prefix: Prefix
-    Location: Location
+    Location: deployment().location
```

Comment:
Can this be grouped into a var called location rather than individual calls to `deployment().location`? There are scenarios where having the ability to change this will be handy, so having it all in a single location would be helpful.